launcher/Application.cpp

```diff
@@ -1516,3 +1519,13 @@ QString Application::getJarsPath()
     }
     return m_jarsPath;
 }
+
+QString Application::getMSAClientID() 
+{
+    auto clientIDOverride = m_settings->get("MSAClientIDOverride").toString();
```

Comment:
this is nitpicky but why are you using auto here but using QString in the other place where you get the setting
```suggestion
    QString clientIDOverride = m_settings->get("MSAClientIDOverride").toString();
```
other than this lgtm